libs/datamodel/parser-database/src/walkers.rs

```diff
@@ -77,6 +77,22 @@ impl ParserDatabase {
             .map(move |ctid| CompositeTypeWalker { ctid, db: self })
     }
 
+    /// Walk all scalar field defaults with a function not part of the common ones.
+    pub fn walk_scalar_field_defaults_with_unknown_function(&self) -> impl Iterator<Item = DefaultValueWalker<'_>> {
+        self.types
+            .unknown_function_defaults
+            .iter()
+            .map(|(model_id, field_id)| DefaultValueWalker {
+                model_id: *model_id,
+                field_id: *field_id,
+                db: self,
+                default: self.types.scalar_fields[&(*model_id, *field_id)]
```

Comment:
Couldn't we just have this as a function in the walker? Is there a reason we store it to the walker? And is the unwrap necessary here?

Reply:
I'd be in favour of removing the `default` field in there, but didn't want to make the PR larger.